daphne_worker_test/tests/test_runner.rs

```diff
@@ -264,6 +264,14 @@ impl TestRunner {
         }
     }
 
+    pub fn max_batch_interval(&self) -> Interval {
```

Comment:
It's not clear what this method is supposed to do based on its signature alone.

* All methods in public APIs should have documentation. This is true no matter what language or codebase.
* This particular method is only used once and is not likely needed elsewhere. I think it's better to just construct the batch interval you need, where you need it.

Reply:
Maybe I'm not understanding it correctly, but this was to resolve this comment (https://github.com/cloudflare/daphne/pull/76#discussion_r940433535).
What do you mean when you say "don't use `t.batch_interval()`" and "construct batch_interval() yourself"?
We need `t.batch_interval()` in any case...

Reply:
I mean construct the batch interval explicitly:
```rust
       batch_interval: Interval {
            start: t.now - t.max_batch_duration / 2,
            duration: t.max_batch_duration,
        },
```
This is much more readable.

Reply:
I've removed the function and directly create batch_interval.